com.unity.shadergraph/Editor/GraphUI/ShaderGraphStencil.cs

```diff
@@ -44,21 +46,44 @@ public override ISearcherFilterProvider GetSearcherFilterProvider()
         }
 
         private Registry.Registry RegistryInstance = null;
+
         public Registry.Registry GetRegistry()
         {
             if (RegistryInstance == null)
             {
-                RegistryInstance = Registry.Default.DefaultRegistry.CreateDefaultRegistry();
+                m_NodeUIHints = new Dictionary<RegistryKey, Dictionary<string, float>>();
```

Comment:
I think this should be initialized in the constructor but filled here to avoid possible NPEs from other methods trying to access it.